### ISSUE TITLE
Download Poster art with modifier?

### ISSUE THREAD

**Issue by joeynotjoe** [vinaywadhwa/justwatch-alfred#4]
Hello! Amazing workflow, thanks for making it! I was wondering, is there an easy way to add in a modifier to download (or copy to clipboard) the poster art?

**Comment by vinaywadhwa** [comment]
Good idea indeed. There just might be a way. I will consider this feature in future updates. Thanks!
